csm_web/frontend/static/frontend/css/policies-styles.css

```diff
@@ -0,0 +1,34 @@
+.sections {
```

Comment:
Regarding the class names, it may be better to be a little more specific than `.sections`, as we ideally do not want any class name conflicts in our codebase—perhaps `.policy-sections` or something similar could be good.

Reply:
Resolved, changed "sections" class to "policy-secitons".